platform-hub-api/spec/controllers/identity_flows_controller_spec.rb

```diff
@@ -2,4 +2,19 @@
 
 RSpec.describe IdentityFlowsController, type: :controller do
 
+  describe "GET #start_auth_flow" do
+    it_behaves_like 'unauthenticated not allowed'  do
+      before do
+        get :start_auth_flow, params: { service: 'github' }
+      end
+    end
+  end
+
+  describe "GET #callback" do
+    it 'should allow unauthenticated access but cannot process the request due to a missing param' do
+      get :callback, params: { service: 'github' }
+      expect(response).to have_http_status(422)
```

Comment:
I'm not sure you can do that in a test? Either way, in tests I much prefer to test the actual status code returned since the symbol aliases are just a Rails thing and can get confusing (e.g. `:unauthorized` and `:forbidden`).